.github/workflows/golangci-lint.yml

```diff
@@ -32,16 +32,7 @@ jobs:
       - name: Install snmp_exporter/generator dependencies
         run: sudo apt-get update && sudo apt-get -y install libsnmp-dev
         if: github.repository == 'prometheus/snmp_exporter'
-      - name: Check for Go file changes
```

Comment:
I keep forgetting about these files lives in the upstream Prometheus. I will propose the changes to that repo.

Reply:
What exactly change you have in mind?

Reply:
Are we ok to merge this PR?